cli-body.js

```diff
@@ -481,6 +480,14 @@ async function main () {
   return files
 }
 
+function getPackageMain (cwd, mainFile = 'index.js') {
+  try {
+    return [path.relative(cwd, require.resolve(path.join(cwd, mainFile)))]
+  } catch (e) {
```

Comment:
What kind of error can we get here?

Reply:
When the file doesn't exist, it throws an cannot find module error:

```sh
Error: Cannot find module '/Users/hwong/code/unional/size-limit/test/fixtures/webpack-multipe-entry-points/index.js'
        at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
        at Function.resolve (internal/modules/cjs/helpers.js:30:19)
        at getPackageMain (/Users/hwong/code/unional/size-limit/cli-body.js:485:40)
        at Promise.all.configFile.config.map (/Users/hwong/code/unional/size-limit/cli-body.js:344:17)
        at Array.map (<anonymous>)
        at main (/Users/hwong/code/unional/size-limit/cli-body.js:335:54)
```

It is observed in the `webpack-multile-entry-points` test.